docs/en/engines/table-engines/mergetree-family/invertedindexes.md

```diff
@@ -48,7 +48,7 @@ CREATE TABLE tab
                                 [, dictionary_block_frontcoding_compression = B]
                                 [, max_cardinality_for_embedded_postings = M]
                                 [, bloom_filter_false_positive_rate = R]
-                            ) [GRANULARITY 64]
+                            )
```

Comment:
The docs say this further down:

```
**Other arguments (optional)**. Text indexes in ClickHouse are implemented as [secondary indexes](/engines/table-engines/mergetree-family/mergetree.md/#skip-index-types).
However, unlike other skipping indexes, text indexes have a default index GRANULARITY of 64.
This value has been chosen empirically and it provides a good trade-off between speed and index size for most use cases.
Advanced users can specify a different index granularity (we do not recommend this).
```

We should say that the
- index granularity is infinite, i.e. the text index is created for the entire part,
- explicitly specified index granularities are ignored.